tests/Support/SupportArrTest.php

```diff
@@ -907,14 +907,30 @@ public function testSet()
         $this->assertEquals([1 => 'hAz'], Arr::set($array, 1, 'hAz'));
     }
 
-    public function testShuffleWithSeed()
+    public function testShuffle()
     {
-        $this->assertEquals(
-            Arr::shuffle(range(0, 100, 10), 1234),
-            Arr::shuffle(range(0, 100, 10), 1234)
+        $input = ['a', 'b', 'c', 'd', 'e', 'f'];
+
+        $this->assertNotEquals(
+            $input,
+            Arr::shuffle($input)
+        );
+
+        $this->assertNotEquals(
+            Arr::shuffle($input),
+            Arr::shuffle($input)
         );
```

Comment:
This is also going to fail in 1 of 720 cases. Combined this gives a failure rate of 1439/518400 for this testcase, which is roughly 1 in 360, which is too high of a false-positive rate for my taste.